tools/bin/goreleaser_utils

```diff
@@ -58,19 +60,76 @@ before_hook() {
   cp -f "$wasmvm_lib_path_darwin_amd64" "$lib_path/darwin_amd64/libs"
   mkdir -p "$lib_path/darwin_arm64/libs"
   cp -f "$wasmvm_lib_path_darwin_arm64" "$lib_path/darwin_arm64/libs"
+
+  # MOVE PLUGINS HERE
+  gobin=$(go env GOPATH)/bin
+
+  install_local_plugins  "linux" "amd64" "$gobin"/linux_amd64
+  install_remote_plugins "linux" "amd64" "$gobin"/linux_amd64
+  mkdir -p "$lib_path/linux_amd64/plugins"
+  cp "$gobin"/linux_amd64/chainlink* "$lib_path/linux_amd64/plugins"
+
+  install_local_plugins  "linux" "arm64" "$gobin"/linux_arm64
+  install_remote_plugins "linux" "arm64" "$gobin"/linux_arm64
+  mkdir -p "$lib_path/linux_arm64/plugins"
+  cp "$gobin"/linux_arm64/chainlink* "$lib_path/linux_arm64/plugins"
+}
+
+install_local_plugins() {
+  local -r goos=$1
+  local -r goarch=$2
+  local -r gobin=$3
+  
+  GOBIN=$gobin GOARCH=$goarch GOOS=$goos make install-medianpoc 
+  GOBIN=$gobin GOARCH=$goarch GOOS=$goos make install-install-ocr3-capability
+}
+
+get_remote_plugin_paths() {
+  plugins=(
+    "github.com/smartcontractkit/chainlink-solana|/pkg/solana/cmd/chainlink-solana"
```

Comment:
What's with this pipe operator?

`ana|/p`

Reply:
its not a pipe, just a separator